transports/jms/src/main/java/org/mule/transport/jms/JmsConnector.java

```diff
@@ -432,7 +432,7 @@ protected Connection createConnection() throws MuleException, JMSException
 
         Connection connection;
 
-        if (username != null)
+        if (!cacheJmsSessions && username != null)
```

Comment:
Yes you can, but user/password are set on CachingConnectionFactory and not passed when requesting connection.  This is the cause of this bug.